pkg/controller/pod.go

```diff
@@ -472,6 +499,24 @@ func addIfaceEventFormat(pod *corev1.Pod, network *nadv1.NetworkSelectionElement
 	)
 }
 
+func failedAddingIfaceEventFormat(pod *corev1.Pod, network *nadv1.NetworkSelectionElement) string {
+	attributes := ""
+	if len(network.IPRequest) > 0 || network.MacRequest != "" || network.CNIArgs != nil {
+		attributes = fmt.Sprintf("(ips: %v, mac: %s, cni-args: %v)",
+			network.IPRequest,
+			network.MacRequest,
+			network.CNIArgs,
+		)
+	}
+	return fmt.Sprintf(
+		"pod [%s]: failed adding interface %s to network: %s%s",
+		annotations.NamespacedName(pod.GetNamespace(), pod.GetName()),
+		network.InterfaceRequest,
+		network.Name,
+		attributes,
+	)
```

Comment:
i might be missing something, but why don't you `Sprintf("...%v", network)` ? 

If we only want to print a subset of the elements, it would be simpler (IMO) to create a new network object w/ just the required data, and format that.

Reply:
Yes, this could be done, but I copied the `addIfaceEventFormat` function and kept the same format.